docs/storefront/headless/routes.mdx

```diff
@@ -0,0 +1,204 @@
+# Routes with the GraphQL Storefront API
+
+Routes define specific URL paths for fetching or updating data. The BigCommerce GraphQL Storefront API allows you to retrieve routes for each storefront channel, making it easier to build headless storefront applications using BigCommerce's built-in content. 
```

Comment:
The `routes` node cannot perform updates. I would rewrite these sentences to something like:

```
The BigCommerce GraphQL Storefront API allows you to retrieve routes for each storefront channel, making it easier to build headless storefront applications using BigCommerce's built-in content. Use of the `route` node is based on a supplied URL. The response will be an entity that exists at that URL, a redirect to a new location, or nothing if the URL does not relate to an entity or redirect.
```